bindings/python/tools/python/ostk/astrodynamics/pytrajectory/pystate.py

```
@@ -100,6 +105,17 @@
         "vz",
     ]
 
+    # Replace non-standard position keys with canonical representation
+    if all(key in data.keys() for key in ("x_eci", "y_eci", "z_eci")):
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)